### PR TITLE
Update server-core-roles-and-services.md

### PR DESCRIPTION
**Pull request by thefaftek**
Updating to indicate that broker is no longer available in 2019 1903.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@thefaftek : Thanks for your contribution! The author(s) have been notified to review your proposed change. @pronichkin

**Comment by PRMerger7** [comment]
@thefaftek : Thanks for your contribution! The author(s) have been notified to review your proposed change. @pronichkin

**Comment by PRMerger10** [comment]
@thefaftek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/administration/server-core/server-core-roles-and-services.md
Comment (illfated) [inline WindowsServerDocs/administration/server-core/server-core-roles-and-services.md:238]: improvement suggestion (readability & formatting):
```suggestion
* Services indicated with an asterisk ( * ) are no longer available in server core starting with Server 2019 1803.
```

File: WindowsServerDocs/administration/server-core/server-core-roles-and-services.md
Comment (illfated) [inline WindowsServerDocs/administration/server-core/server-core-roles-and-services.md:76]: formatting: we like to keep the table line structure intact
```suggestion
| Remote Desktop Services               | Remote Desktop Connection Broker*                              | RDS-Connection-Broker   | N                     |
```

**Review by illfated — COMMENTED** [review]
I do not have the technical knowledge to confirm this change, so I will be waiting for approval by the assigned author.

**Comment by illfated** [comment]
The page author has confirmed the added statement to be correct, in issue ticket #5011. Looks like I can approve the content after the suggestions are applied.

**Comment by Karl-WE** [comment]
Hi @thefaftek nice to meet you over here.
Once your PR is merge I will create a new PR containing these.
https://github.com/MicrosoftDocs/windowsserverdocs/issues/3016

**Comment by Karl-WE** [comment]
Hello Artem @pronichkin do you think this one can be signoff ?
How do you prefer that I should include the changes of #3016. Would like to wait for this PR to signoff and merge before open a new one to avoid clutter.

**Review by pronichkin — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Fair enough. My suggestions are nearly cosmetic, compared to the information update.

Looks good to me.

**Comment by illfated** [comment]
@eross-msft : May we have your advice with regards to moving forward with this PR?
Looks like @pronichkin does not have merging rights in this doc or repo.

**Comment by eross-msft** [comment]
@Karl-WE @illfated  I'm going to attempt to merge these changes; however, our publishing pipeline is currently broken, so I'm not sure if it will actually work. It fails "gracefully", which means it LOOKS like it worked, but the changes never come through. If you still don't see your updates in the live content by tomorrow morning, please let me know and I'll try to re-publish. Thanks!!!

**Comment by illfated** [comment]
Thank you for the feedback & information update. Good to know. I may recreate my remaining PRs in a week or so, to be ready when the publishing pipeline gets restored.

**Comment by thefaftek** [comment]
Thanks for the updates, review, and approval on this! Sorry on the
formatting, was still learning how this all worked at the time!

**Pull request merged**
